src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java

```diff
@@ -83,15 +84,23 @@ public void currentFrameShouldNotContain(String text, String...params) {
 	@RobotKeyword("Verify the element identified by ``locator`` contains ``text``.\n\r"
 	        + "\n\r"
 	        + "See `Introduction` for details about locators.")
-	@ArgumentNames({ "locator", "text", "message=NONE" })
+	@ArgumentNames({ "locator", "text", "message=NONE", "ignore_case=False" })
 	public void elementShouldContain(String locator, String text, String...params) {
-	    String message = robot.getParamsValue(params, 0, "");
+        String message = robot.getParamsValue(params, 0, "");
```

Comment:
As we use StringUtils from commons-lang3, perhaps, we can use [EMPTY](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#EMPTY) instead of `""` here.

Reply:
Good idea.